### DISCUSSION TITLE
Start Frame and "0".

### DISCUSSION THREAD

**Discussion by ethanfel** [FizzleDorf/ComfyUI_FizzNodes#55] [Q&A] [answered]
First, thank you for your awesome node.

I have just a question, does using a number in the start frame move the "0" or does it still apply to the first frame ?

For example "0" : "closed eyes", without start frame number will apply to the first frame.
If I set start frame to 8, does closed eyes apply to the 8 frame or does it stay at 0 ?

**Comment by FizzleDorf** [comment, marked as answer]
hello @ethanfel, the start_frame is for debugging print outputs only. start frame is only for which frame to start the print in the cli for batch scheduling operations.
